apps/android/LiveVideoApp/app/src/main/java/com/twilio/livevideo/app/repository/model/GenericResponse.kt

```diff
@@ -0,0 +1,42 @@
+package com.twilio.livevideo.app.repository.model
+
+import com.google.gson.annotations.SerializedName
+
+data class GenericResponse(
+
+    @SerializedName("verified")
+    val isVerified: Boolean? = null,
+
+    @SerializedName("success")
+    val isSuccess: Boolean? = null,
+
+    @SerializedName("sent")
+    val isSent: Boolean? = null,
+
+    @SerializedName("removed")
+    val isRemoved: Boolean? = null,
+
+    @SerializedName("token")
+    val token: String? = null
+
+) : BaseResponse()
+
+data class ErrorResponse(
+
+    @SerializedName("message")
+    val message: String? = null,
+
+    @SerializedName("explanation")
+    val explanation: String? = null
```

Comment:
The backend API is such that these will never be null. Would you still make them optional in Android?

Reply:
Oh no, actually I tested the api calls using Postman, and... if the response is success, then the error body is coming `null`.